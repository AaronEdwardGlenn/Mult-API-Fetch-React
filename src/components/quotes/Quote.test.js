import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote component', () => {
  it('quote matches the snap', () => {
    const wrapper = shallow(<Quote text="calvin" name="coolidge" img="yah/brah/img" />);
    expect(wrapper).toMatchSnapshot();
  });
})
;
