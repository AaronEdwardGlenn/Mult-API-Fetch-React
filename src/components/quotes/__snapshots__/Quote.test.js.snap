// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component quote matches the snap 1`] = `
<figure>
  <img
    src="yah/brah/img"
  />
  <figcaption>
    "
    calvin
    " 
    -- by coolidge
  </figcaption>
</figure>
`;
